capstone/src/main/java/com/google/step/clients/MetricsClient.java

```diff
@@ -116,7 +116,7 @@ public WeeklyPageView getCurrentPageViews(String restaurantKey) {
      * @return returns List<RestaurantPageViews> with the PageView list inside each instance
      *         of RestaurantPageViews sorted by week and year in ascending order.
      */
-    public List<RestaurantPageViews> getAllPageViews() {
+    public List<RestaurantPageViews> getAllPageViews(String identificationType) {
```

Comment:
Why did you add this parameter?

Reply:
I added the parameter because Josh needed to have the restaurantKey instead of the restaurantName for the scoring stuff. That way I could choose which one I got depending on the need.

Reply:
I'd prefer we add an additional field for the restaurant key to the RestaurantPageViews for the following reason:
1. The documentation and function/field names of the RestaurantPageViews class do not work well with this set-up. For example, will Josh need to call getName() to get the restaurant key? I think it would be easier to use and understand if there was a new restaurantKey field and getRestaurantKey method.
2. We already have the entity, so there's no significant performance cost to populating the RestauarantPageViews object with both values (e.g. no need for an extra query).

Reply:
Yeah that does make sense! Changed!